src/main/default/lwc/lookup/lookup.js

```diff
@@ -263,6 +267,17 @@ export default class Lookup extends LightningElement {
         this._hasFocus = false;
     }
 
+    handleOnClick(evt) {
+        if (!this.isSelectionAllowed()) {
```

Comment:
let's look into this together, maybe we can find a way that does not change too much in the component and is not using attributes like focus that are not return the true focus status